reframe/frontend/cli.py

```diff
@@ -197,26 +197,21 @@ def main():
     # Run options
     run_options.add_argument(
         '-A', '--account', action='store',
-        help='Use ACCOUNT for submitting jobs (Slurm)'
-    )
+        help='*deprecated*, please use --job-option instead')
```

Comment:
We should keep the help message and add also the deprecation highlight saying how to replace it.

```suggestion
        help="*deprecated*, please use '-J account=ACCOUNT'")
```

Reply:
At this point we can't know which scheduler is being used, right? If we put `-J account=ACCOUNT`, it would be slurm syntax, even if the user is running PBS. That's why I preferred a generic message.

Reply:
I'd put both: `-J account=NAME or -J q=NAME`. To save space you can also set `metavar='NAME'` so that you can use that name in your help message.